Installing plugin causes build failures

Ran the install:
`yarn add --cwd packages/backend @mdude2314/backstage-plugin-scaffolder-git-actions`

Added the typescript code: (packages/backend/src/plugins/scaffolder.ts)
```
import { CatalogClient } from '@backstage/catalog-client';
import { createRouter } from '@backstage/plugin-scaffolder-backend';
import { Router } from 'express';
import type { PluginEnvironment } from '../types';

// Added This
import { gitCloneAction, gitAction } from "@mdude2314/backstage-plugin-scaffolder-git-actions";

export default async function createPlugin(
  env: PluginEnvironment,
): Promise<Router> {
  const catalogClient = new CatalogClient({
    discoveryApi: env.discovery,
  });

  // Added This
  const builtInActions = createBuiltinActions({
    catalogClient,
    integrations,
    config: env.config,
    reader: env.reader
  });

  // Added This
  const actions = [
    gitAction(),
    gitCloneAction(),
    ...builtInActions
  ];

  return await createRouter({
    logger: env.logger,
    config: env.config,
    database: env.database,
    reader: env.reader,
    catalogClient,
    // Added This
    actions
  });
}
```

Ran test:
```
yarn tsc
yarn run v1.22.19
$ tsc

packages/backend/src/plugins/scaffolder.ts:15:26 - error TS2304: Cannot find name 'createBuiltinActions'.

15   const builtInActions = createBuiltinActions({
                            ~~~~~~~~~~~~~~~~~~~~

packages/backend/src/plugins/scaffolder.ts:17:5 - error TS18004: No value exists in scope for the shorthand property 'integrations'. Either declare one or provide an initializer.

17     integrations,
       ~~~~~~~~~~~~


Found 2 errors in the same file, starting at: packages/backend/src/plugins/scaffolder.ts:6

error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

My packages/backend/package.json
```
{
  "name": "backend",
  "version": "0.0.0",
  "main": "dist/index.cjs.js",
  "types": "src/index.ts",
  "private": true,
  "backstage": {
    "role": "backend"
  },
  "scripts": {
    "start": "backstage-cli package start",
    "build": "backstage-cli package build",
    "lint": "backstage-cli package lint",
    "test": "backstage-cli package test",
    "clean": "backstage-cli package clean",
    "build-image": "docker build ../.. -f Dockerfile --tag backstage"
  },
  "dependencies": {
    "@backstage/backend-common": "^0.14.1",
    "@backstage/backend-tasks": "^0.3.3",
    "@backstage/catalog-client": "^1.0.4",
    "@backstage/catalog-model": "^1.1.0",
    "@backstage/config": "^1.0.1",
    "@backstage/plugin-app-backend": "^0.3.34",
    "@backstage/plugin-auth-backend": "^0.15.0",
    "@backstage/plugin-catalog-backend": "^1.3.0",
    "@backstage/plugin-jenkins-backend": "^0.1.25",
    "@backstage/plugin-permission-common": "^0.6.3",
    "@backstage/plugin-permission-node": "^0.6.3",
    "@backstage/plugin-proxy-backend": "^0.2.28",
    "@backstage/plugin-scaffolder-backend": "^1.4.0",
    "@backstage/plugin-search-backend": "^1.0.0",
    "@backstage/plugin-search-backend-module-pg": "^0.3.5",
    "@backstage/plugin-search-backend-node": "^1.0.0",
    "@backstage/plugin-sonarqube-backend": "^0.1.0",
    "@backstage/plugin-techdocs-backend": "^1.2.0",
    "@mdude2314/backstage-plugin-scaffolder-git-actions": "^1.1.0",
    "app": "link:../app",
    "dockerode": "^3.3.1",
    "express": "^4.17.1",
    "express-promise-router": "^4.1.0",
    "pg": "^8.7.3",
    "winston": "^3.2.1"
  },
  "devDependencies": {
    "@backstage/cli": "^0.18.0",
    "@types/dockerode": "^3.3.0",
    "@types/express": "^4.17.6",
    "@types/express-serve-static-core": "^4.17.5",
    "@types/luxon": "^2.0.4",
    "better-sqlite3": "^7.5.0"
  },
  "files": [
    "dist"
  ]
}
```

--- a/README.md
+++ b/README.md
@@ -7,6 +7,9 @@ This contains a collection of actions for using with git:
 - git
 - git:clone
 
+## Prerequisites
+- Git must be installed in the environment your Backstage instance is running in
+
 ## Getting started
 
 In the root directory of your Backstage project:
@@ -21,8 +24,12 @@ Add the actions you'd like to the scaffolder:
 // packages/backend/src/plugins/scaffolder.ts
 
 import { gitCloneAction, gitAction } from "@mdude2314/backstage-plugin-scaffolder-git-actions";
+import { ScmIntegrations } from '@backstage/integration';
+import { createBuiltinActions, createRouter } from '@backstage/plugin-scaffolder-backend';
+
 ...
 
+const integrations = ScmIntegrations.fromConfig(env.config);
 const builtInActions = createBuiltinActions({
   catalogClient,
   integrations,

--- a/README.md
+++ b/README.md
@@ -87,9 +87,9 @@ spec:
       name: git
       action: git
       input:
-        command: ${{ parameters.command }}
-        workingDirectory: ${{ parameters.workingDirectory }}
-        args: ${{ parameters.args }}
+        command: ${{ parameters.command }} # ex: 'commit' - will make the scaffolder run the `git commit` command
+        workingDirectory: ${{ parameters.workingDirectory }} # ex: './my-working-directory' - will execute the command in the specified directory relative to the scaffolder workspace
+        args: ${{ parameters.args }} # ex: ['-m', 'My commit message'] - will add '-m My commit message' to the arguments passed to the git command
 ```
 
 ## Example of using the clone action

